app/views/includes/result-item.nunjucks

```diff
@@ -1,6 +1,6 @@
 <h3 class="results__name">
-  {% if gp.supplier %}
-    <a href="{{gp.supplier}}">{{ gp.name }}</a>
+  {% if gp.bookingSystem %}
+    <a href="{{gp.bookingSystem.bookOnlineLink}}">{{ gp.name }}</a>
```

Comment:
This is optional. Suggest adding the check to the line above e.g. `{% if gp.bookingSystem && gp.bookingSystem.bookOnlineLink %}`